### DISCUSSION TITLE
花篮接力统计及讨论

### DISCUSSION THREAD

**Discussion by Misaka-0x447f** [project-trans/basket-relay#2] [General]
![camphoto_959030623](https://user-images.githubusercontent.com/15797507/230281139-87b1d61c-6fac-4f0a-8754-608b3cf6c99b.jpeg)

总觉得不记录下来实在太可惜了，于是开一个实验性的 issue。
该花篮最初是 F.Casta Misaka 购买给 洋芋 Efina 的。
我会打印一个二维码贴在花篮上。
该花篮不是专属于某个医院的。可以随意自行选择传递对象。但建议具备基本的 Github discussions 使用能力。

**Comment by BingL-Li** [comment]
我从 kiwi🥝那边接手，给了 カラスのばかねこ

**Comment by leohearts** [comment]
来了。你们还真的做成传家宝 repo 了啊 (躺

**Reply by Misaka-0x447f** [reply]
甚至还去 ikea 又买了点花。

**Comment by Misaka-0x447f** [comment]
@Misaka-0x447f 2023/03/20 & Fuzhu 2023/03/21
暂时拜托给 🌸 保管

**Comment by tuboshufps** [comment]
我从Gekka Saori接手
大概是从去年9月？开始传，一直传到现在吧

**Comment by mcfx** [comment]
![IMG_3321](https://user-images.githubusercontent.com/25007037/230416613-471f9f41-639d-44b4-86ca-a6df0b5c9fa6.jpeg) 接手了花篮（

**Comment by tuboshufps** [comment]
o/ again. any idea where is it or who has it RN? :)

**Comment by Misaka-0x447f** [comment]
> o/ again. any idea where is it or who has it RN? :)

Seems already lost.

**Comment by tuboshufps** [comment]
已经有一年了，据说目前在WIH 7F前台

**Comment by tuboshufps** [comment]
证实^

**Comment by Starrah** [comment]
我（星露）于2023.8.14在WIH从小狗处接过此花篮。
会好好保存并交给下一个人的。
Update: 已于2023.8.25交给鸠。
![IMG_20230815_174845](https://github.com/project-trans/reports/assets/46821571/b2a60a1d-ecb3-4a5f-a5a7-4820653330e1)

**Comment by JIUJIUHIME** [comment]
鸠于8.25收到花篮，于9.15传递给最废的比（910号房)。

**Comment by 1165815115** [comment]
最废的比（910）于2023年9月24日将花篮传给kelili。kelili给花篮喷了点花香水，使它变的香香哒～

**Comment by Misaka-0x447f** [comment]
？还活着啊

**Reply by misakayin** [reply]
2023年10月曾传递到我(御坂)901房这里

**Reply by misakayin** [reply]
当时过于疼痛，以至于忘记回复惹

**Comment by 1165815115** [comment]
还能这么玩？



---原始邮件---
发件人: "MisakaKumomi ***@***.***&gt;
发送时间: 2023年10月24日(周二) 中午1:24
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [project-trans/reports] 花篮接力统计及讨论 (Discussion #2)




 
？还活着啊

**Comment by 1165815115** [comment]
花篮在我这里，我明天就离开wih

**Reply by 1165815115** [reply]
我会把所以东西寄存在护士台，直到下一个中国人来

**Comment by 1165815115** [comment]
我会留下wih的QQ群号

**Comment by 1165815115** [comment]
已经移交9楼护士台

**Reply by Misaka-0x447f** [reply]
感谢

**Comment by Misaka-0x447f** [comment]
Okay that's it. 可能是二维码遗失，也可能是就这样了。毕竟我用的是热敏纸打印。
